test/dlc_tests/ecs/tensorflow/training/test_ecs_tensorflow_training.py

```diff
@@ -42,7 +42,7 @@ def test_ecs_tensorflow_training_mnist_cpu(
 @pytest.mark.model("mnist")
 @pytest.mark.team("frameworks")
 @pytest.mark.parametrize("training_script", [TF_MNIST_TRAINING_SCRIPT], indirect=True)
-@pytest.mark.parametrize("ecs_instance_type", ["p3.8xlarge"], indirect=True)
+@pytest.mark.parametrize("ecs_instance_type", ["g4dn.8xlarge"], indirect=True)
```

Comment:
Was this just to fix ICE? Is this a permanent change?

Reply:
This is permanent. The problem isn't ICE, but rather the fact that the instance termination just fails to happen within a reasonable amount of time, causing the boto client waiter to timeout and fail the test.